api_tests/nodes/views/test_node_forks_list.py

```diff
@@ -10,8 +11,11 @@
     ForkFactory
 )
 from rest_framework import exceptions
+from website import mails
 from website.util import permissions
 
+from api.nodes.serializers import NodeForksSerializer
+mails.send_mail = mock.MagicMock()
```

Comment:
This is consistent with how the MagicMock is used on the rest of the site, this isn't really a patch per say.